WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -804,113 +781,4 @@ protected void onPostExecute(SiteList[] updatedSiteLists) {
             mDataLoadedListener.onAfterLoad();
         }
     }
-
-    /**
-     * SiteRecord is a simplified version of the full account (blog) record
-     */
-    public static class SiteRecord {
-        private final int mLocalId;
-        private final long mSiteId;
-        private final String mBlogName;
-        private final String mHomeURL;
-        private final String mBlavatarUrl;
-        private final ImageType mBlavatarType;
-        private boolean mIsHidden;
-        private boolean mIsRecentPick;
-
-        public SiteRecord(SiteModel siteModel) {
-            mLocalId = siteModel.getId();
-            mSiteId = siteModel.getSiteId();
-            mBlogName = SiteUtils.getSiteNameOrHomeURL(siteModel);
-            mHomeURL = SiteUtils.getHomeURLOrHostName(siteModel);
-            mBlavatarUrl = SiteUtils.getSiteIconUrl(siteModel, mBlavatarSz);
-            mBlavatarType = SiteUtils.getSiteImageType(
-                    siteModel.isWpForTeamsSite(), BlavatarShape.SQUARE_WITH_ROUNDED_CORNERES);
-            mIsHidden = !siteModel.isVisible();
-        }
-
-        public String getBlogNameOrHomeURL() {
-            if (TextUtils.isEmpty(mBlogName)) {
-                return mHomeURL;
-            }
-            return mBlogName;
-        }
-
-        public int getLocalId() {
-            return mLocalId;
-        }
-
-        public boolean isHidden() {
-            return mIsHidden;
-        }
-
-        public void setHidden(boolean hidden) {
-            mIsHidden = hidden;
-        }
-
-        public String getHomeURL() {
-            return mHomeURL;
-        }
-
-        public String getBlavatarUrl() {
-            return mBlavatarUrl;
-        }
-
-        public ImageType getBlavatarType() {
-            return mBlavatarType;
-        }
-
-        public long getSiteId() {
-            return mSiteId;
-        }
-    }
-
-    public static class SiteList extends ArrayList<SiteRecord> {
```

Comment:
Simply use `List<SiteRecord>` instead of `SiteList` which is easier to understand.